src/main/webapp/app/oncokb-commons/components/RefComponent.tsx

```diff
@@ -0,0 +1,62 @@
+import { getNCBIlink } from './lib/urls';
+
+export type ParsedRef = {
+  prefix?: string;
+  content: string;
+  link?: string;
+  isAbstract?: boolean;
+};
+
+export const parseReferences = (content: string, seperatePMIDs = false): ParsedRef[] => {
```

Comment:
I am trying to extract the logic from [here](https://github.com/oncokb/oncokb-public/blob/master/src/main/webapp/app/oncokb-frontend-commons/src/components/RefComponent.tsx) to a function. The code from oncokb-public frontend commons returns a component, but I just need to parsing logic for my use case.

I think I should probably move this to a util file instead of under `oncokb-commons` folder.